--- "node_modules\\redux-persist\\types\\persistStore.d.ts"	1985-10-26 10:15:00.000000000 +0200
+++ "patch\\persistStore.d.ts"	2020-09-02 11:02:45.725461600 +0200
@@ -1,5 +1,5 @@
 declare module "redux-persist/es/persistStore" {
-  import { Store } from 'redux';
+  import { Store, Action, AnyAction } from 'redux';
   import { PersistorOptions, Persistor } from "redux-persist/es/types";
 
   /**
@@ -9,7 +9,7 @@
    * @param callback bootstrap callback of sort.
    */
   // tslint:disable-next-line: strict-export-declare-modifiers
-  export default function persistStore(store: Store, persistorOptions?: PersistorOptions | null, callback?: () => any): Persistor;
+  export default function persistStore<S = any, A extends Action<any> = AnyAction>(store: Store<S, A>, persistorOptions?: PersistorOptions | null, callback?: () => any): Persistor;
 }
 
 declare module "redux-persist/lib/persistStore" {
